backend/app.py

```diff
@@ -177,6 +236,40 @@ def get_steam_level():
             return 'failure to get steam level games. Error: HTTP {}'.format(response.status_code)
     else:
         return "invalid steamid"
+      
+   
+@app.route("/api/getplayersummary", methods=['POST'])
```

Comment:
We have two duplicates. In a future story, lets refactor/remove one of them. (No pending changes for now)